installer/datafiles/base_container.data

```diff
@@ -170,6 +174,14 @@ touch /var/opt/microsoft/docker-cimprov/log/kubernetes_perf_log.txt
 chmod 666 /var/opt/microsoft/docker-cimprov/log/kubernetes_perf_log.txt
 chown omsagent:omiusers /var/opt/microsoft/docker-cimprov/log/kubernetes_perf_log.txt
 
+touch /var/opt/microsoft/docker-cimprov/log/filter_cadvisor2mdm.log
+chmod 666 /var/opt/microsoft/docker-cimprov/log/filter_cadvisor2mdm.log
```

Comment:
can you please xplain why you are touching here ? Hos is thsi plugin different from other plugins ? Also why would you need write prems (i see 666)...